manim/mobject/geometry.py

```diff
@@ -863,6 +863,119 @@ def __init__(self, n=6, **kwargs):
         Polygon.__init__(self, *vertices, **kwargs)
 
 
+class ArcPolygon(VMobject):
+    """
+    A more versatile polygon, made from arcs.
+
+    For proper appearance the passed arcs should seamlessly connect:
+    [a,b][b,c][c,a]
+
+    If there are any gaps between the arcs, those will be filled in
+    with straight lines, which can be used deliberately for any straight
+    sections. Arcs can also be passed as straight lines such as an arc
+    initialized with 'angle=0'.
+
+    ArcPolygons can be transformed properly into one another as well. Be
+    advised that any arc initialized with 'angle=0' will actually be
+    made like a line, so if a straight secion should seamlessly
+    transform into an arced section, initialize the straight section
+    with a negligible angle instead (such as 'angle=0.0001').
+
+    Parameters
+    ----------
+    *arcs : Union[:class:`Arc`, :class:`ArcBetweenPoints`]
+        These are the arcs from which the arcpolygon is assembled.
+    **kwargs : Any
+        Affects how the ArcPolygon itself is drawn, but doesn't affect
+        passed arcs.
+
+    Attributes
+    ----------
+    arcs : :attr:`list`
+        The arcs used to initialize the ArcPolygon.
+
+    Examples
+    --------
+    One example of an arcpolygon is the Reuleaux triangle.
+    Instead of 3 straight lines connecting the outer points,
+    a Reuleaux triangle has 3 arcs connecting those points,
+    making a shape with constant width.
+
+    Passed arcs are stored as submobjects in the ArcPolygon.
+    This means that the arcs are changed along with the ArcPolygon,
+    for example when it's shifted, and these arcs can be manipulated
+    after the ArcPolygon has been initialized.
+
+    Also both the arcs contained in an ArcPolygon, as well as the
+    ArcPolygon itself are drawn, which affects draw time in ShowCreation
+    for example. In most cases the arcs themselves don't
+    need to be drawn, in which case they can be passed as invisible.
+        .. manim:: ArcPolygonExample
+
+            class ArcPolygonExample(Scene):
+                def construct(self):
+                    arc_conf = {"stroke_width":0}
+                    poly_conf = {"stroke_width":10,"stroke_color":BLUE,
+                      "fill_opacity":1,"color": PURPLE}
+                    a=[-1,0,0]
+                    b=[1,0,0]
+                    c=[0,np.sqrt(3),0]
+                    arc0=ArcBetweenPoints(a,b,radius=2,**arc_conf)
+                    arc1=ArcBetweenPoints(b,c,radius=2,**arc_conf)
+                    arc2=ArcBetweenPoints(c,a,radius=2,**arc_conf)
+                    reuleaux_tri=ArcPolygon(arc0,arc1,arc2,**poly_conf)
+                    self.play(FadeIn(reuleaux_tri))
+                    self.wait(2)
```

Comment:
This block should not be indented. `.. manim` should start at the same height as `need` in the line above it. (This causes the blue box in the documentation.)

Reply:
So you said previously. I tried it, and all it did was breaking the videos completely. Hence I restored the indents.

Reply:
Because you were missing the blank line before the `.. manim::` statement as well. I've pushed 82ea847a, there the docs compile correctly for me locally.

Reply:
Yeah. That's what I thought too. And then I quickly changed that, pushed it, and... nothing happened.
Or to be more precise, the docs built correctly, but I never got to see the updated version so I merely thought nothing happened. Well, I'm aware of the cursed DNS or whatever it is now.

Reply:
Currently we are dealing with #719, docs will not update until this is fixed.

Reply:
Well RIP, but duly noted.